ee/modules/030-cloud-provider-openstack/images/cloud-data-discoverer/discoverer.go

```diff
@@ -301,7 +301,7 @@ func (d *Discoverer) getImages(ctx context.Context, provider *gophercloud.Provid
 		imageNames = append(imageNames, image.Name)
 	}
 
-	return imageNames, nil
+	return removeDuplicates(imageNames), nil
```

Comment:
We have `set` package in go_lib which make all this in single line. 

fyi @artbin @name212